CombinatorialGames/IGame/Concrete.lean

```diff
@@ -71,6 +72,20 @@ theorem leftMoves_toIGame (a : α) : (toIGame a).leftMoves = toIGame '' {b | b 
 theorem rightMoves_toIGame (a : α) : (toIGame a).rightMoves = toIGame '' {b | b ≺ᵣ a} := by
   rw [toIGame_def, rightMoves_ofSets]
 
+theorem rel_leftMove_iff {a b : α} :
+    toIGame b ∈ (toIGame a).leftMoves ↔ ∃ c, c ≺ₗ a ∧ toIGame c = toIGame b := by
+  simp_rw [leftMoves_toIGame, Set.mem_image, Set.mem_setOf_eq]
```

Comment:
I don't think we need this lemma, since this works:
```suggestion
theorem rel_leftMove_iff {a b : α} :
    toIGame b ∈ (toIGame a).leftMoves ↔ ∃ c, c ≺ₗ a ∧ toIGame c = toIGame b := by
  rw [leftMoves_toIGame]; rfl
```